src/CarbonAware.LocationSources/src/Configuration/LocationDataSourcesConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+namespace CarbonAware.LocationSources.Configuration;
+
+/// <summary>
+/// A configuration class for holding Location Data config values.
+/// </summary>
+public class LocationDataSourcesConfiguration
+{
+
+    public const string Key = "LocationDataSourcesConfiguration";
+
+    public List<LocationDataSource>? LocationDataSources { get; set; }
```

Comment:
Can this be non-nullable instead to clean up the checks downstream?

```suggestion
    public List<LocationDataSource> LocationDataSources { get; set; } = new()
```